docs/tutorial.md

```diff
@@ -0,0 +1,138 @@
+
+```javascript
+const express = require('express')
+const app = express()
+const port = process.env.PORT
+
+app.get('/', (req, res) => {
+  res.send('Hello World!')
+})
+
+app.listen(port, () => {
+  console.log(`Example app listening at http://localhost:${port}`)
+})
+```
+
+4. Define the `Dockerfile` like this: 
+
+```bash
+FROM node:lts
+
+ENV PORT 8088
+
+COPY . .
+
+RUN npm install
+CMD node index.js
+```
+
+5. Test the `Dockerfile` locally by installing [Docker Desktop](https://www.docker.com/products/docker-desktop), building the Docker image and then running the server within the container
+
+```bash
+$ docker build -t anycloud/app .
+$ docker run -p 8088:8088 -d anycloud/app:latest
+$ curl localhost:8088
+```
+
+Which should return `Hello World!`
+
+ 6. Define your deployment configuration in `anycloud.json` like this:
+
+```javascript
+{
+  "staging": [{
+    "credentials": "aws-personal",
+    "region": "us-west-1",
+    "vmType": "t3.medium"
+  }]
+}
+```
+
+We are referencing the previously defined credentials so make sure that the `credentials` value matches the key in `~/.anycloud/credentials.json`
+
+## Deploy an app
+
+1. Make sure you [install the AnyCloud CLI](about.md#cli-installation). Now deploy your Node.js server to your AWS account using the AnyCloud CLI:
+
+```bash
+$ anycloud new staging
+▇ Creating new app
+```
+
+It might take a few minutes for your app to start while the virtual machine is provisioned and upgraded. The AnyCloud CLI will generate a random app name for you, but [a custom app name can be used](how-to-guides/custom-name.md). 
+
+2. Check the status of your application:
+
+```bash
+$ anycloud info
+Status of all apps deployed:
+
+┌─────────────────┬─────────────────────────────────────────┬───────────────┬──────┬─────────┐
+│ App Id          │ Url                                     │ Deploy Config │ Size │ Version │
+├─────────────────┼─────────────────────────────────────────┼───────────────┼──────┼─────────┤
+│ maroon-egret-25 │ https://maroon-egret-25.anycloudapp.com │ staging       │ 1    │ v0.1.34 │
+└─────────────────┴─────────────────────────────────────────┴───────────────┴──────┴─────────┘
+
+Deployment configurations used:
+
+┌───────────────┬──────────────┬────────────────┬────────────┬───────────┐
+│ Deploy Config │ Credentials  │ Cloud Provider │ Region     │ VM Type   │
+├───────────────┼──────────────┼────────────────┼────────────┼───────────┤
+│ staging       │ aws-personal │ GCP            │ us-west-1  │ t3.medium │
+└───────────────┴──────────────┴────────────────┴────────────┴───────────┘
+```
+
+3. The `size` of your app represents the number of virtual machines used to back your app. Apps scale elastically based on request load automatically. Now `curl` your AnyCloud app!
+
+```bash
+$ curl https://maroon-egret-25.anycloudapp.com
+```
+
+Which should return `Hello World!`
+
+4. Terminate your AnyCloud app after you are done
```

Comment:
"Terminate your AnyCloud app when you no longer need it"? The current phrasing somehow makes it seem like they're supposed to be temporary?